pkg/auth/auth.go

```diff
@@ -4,9 +4,11 @@ import (
 	"fmt"
 	"time"
 
+	adal "github.com/Azure/go-autorest/autorest/adal"
```

Comment:
we can probably remove the package name for this import - `"github.com/Azure/go-autorest/autorest/adal"` should be sufficient